### PR TITLE
Add 2021-01-15 meeting notes

### PR DESCRIPTION
**Pull request by matteodelabre**

### CODE REVIEW

File: meetings/discussion-2021-01-15.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        - Find the right balance
+    - Move issues from toltec-dev/toltec
+- Standard name
+    - [@LinusCDE](https://github.com/LinusCDE): Ask people for opinions about this for the next meeting
+    - Ask in the #matrix channel
+    - No name needed if we go with the XDG Desktop format?
+
+# Decision log
+
+| Description             | Decision               |
```
Comment (Eeems) [inline meetings/discussion-2021-01-15.md:89]: No notes column? Do we want to also remove that column from the template?

Comment (matteodelabre) [reply meetings/discussion-2021-01-15.md:89]: I didn’t see anything that fitted into this column. I see you provided in the template an example of what might go into that column. But it seems to me that we can record who voted for and against the decision in the `Decision` column. Did you have something else in mind about the content of these two columns?

Comment (Eeems) [reply meetings/discussion-2021-01-15.md:89]: The sample on the template was saying that "Eeems abstained". Likely something we don't really need though as the decision column can contain this.

Comment (matteodelabre) [reply meetings/discussion-2021-01-15.md]: Okay, I updated the template. Also took the opportunity to convert line endings from DOS to Unix.

Comment (Eeems) [reply meetings/discussion-2021-01-15.md]: Sorry about that, I was working on windows. Maybe we should add a .gitattributes file to force it.

Comment (matteodelabre) [reply meetings/discussion-2021-01-15.md]: Good idea, I added it to this PR.

**Review by Eeems — DISMISSED** [review]
lgtm. We should probably have everyone sign off on it before a merge though.

**Review by LinusCDE — APPROVED** [review]
lgtm

**Comment by Eeems** [comment]
Whoops, edited the wrong PR.

**Review by raisjn — APPROVED** [review]

**Review by Eeems — APPROVED** [review]

**Pull request merged**